src/frontend/src/instance/distributed.rs

```diff
@@ -143,6 +150,19 @@ impl DistInstance {
                     .context(error::ExecuteSqlSnafu { sql })?;
                 self.query_engine.execute(&plan).await
             }
+            Statement::CreateDatabase(stmt) => {
+                let expr = CreateDatabaseExpr {
+                    database_name: stmt.name.to_string(),
+                };
+                self.handle_create_database(expr).await?;
+                Ok(Output::AffectedRows(1))
+            }
+            Statement::CreateTable(stmt) => {
+                let create_expr = &mut DefaultCreateExprFactory {}
```

Comment:
Looks really strange, why not keep a global instance for `DefaultCreateExprFactory`?